src/main/java/uk/gov/gchq/palisade/resource/request/GetResourcesBySerialisedFormatRequest.java

```diff
@@ -39,21 +40,23 @@ public GetResourcesBySerialisedFormatRequest() {
      * @return the {@link GetResourcesBySerialisedFormatRequest}
      */
     public GetResourcesBySerialisedFormatRequest serialisedFormat(final String serialisedFormat) {
```

Comment:
This needs the @generated annotation on the mutator

Reply:
Done